doc/notebooks_classification.rst

```diff
@@ -1,13 +1,10 @@
-Classification Notebooks
+Classification notebooks
 ========================
 
 The following examples present you advanced analyses on multi-class classification
```

Comment:
present you advanced -> present advanced

Reply:
Done